packages/ui/form/PhoneInput.tsx

```diff
@@ -59,17 +58,13 @@ function BasePhoneInput({ name, className = "", onChange, ...rest }: PhoneInputP
 
 const useDefaultCountry = () => {
   const [defaultCountry, setDefaultCountry] = useState("us");
-  trpc.viewer.public.countryCode.useQuery(undefined, {
-    refetchOnWindowFocus: false,
-    refetchOnReconnect: false,
-    retry: false,
-    onSuccess: (data) => {
-      if (isSupportedCountry(data?.countryCode)) {
```

Comment:
we can add a condition here if data?.countryCode === "" (this happens when vercel headers are present with countryCode)  and then we use 
const userCountryCode = navigator.language.split("-")[1]